editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,43 @@
 caption: Formatting
 created: 20131205155959399
-modified: 20220513120211686
+modified: 20221025135601649
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
-Available character formatting includes:
+!! Overview
 
-* <code>&#96;code&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>), alternatively <code>&#96;&#96;double backticks allow &#96;embedded&#96; single backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold'' text (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic// text (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored__ text (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)<br><br>
-* `@@highlight@@` for @@highlighted@@ text (no editor toolbar button), see [[Styles and Classes in WikiText]] for more formatting options
+Available character formatting in WikiText includes:
 
-See also: [[Code Blocks in WikiText]]
+|Wikitext | Button | Shortcut |Rendered Output |h
+|2 single hyphens are used for `''bold text''`| <kbd><<.icon $:/core/images/bold>></kbd> | <kbd><<displayshortcuts "((bold))">></kbd> |2 single hyphens are used for ''bold text'' |
```

Comment:
It's a subtlety that hasn't made it into our docs style guide, but here we should spell out numbers rather than using digits. In this case, I think we should use "Double hyphens" etc., not "two".

https://www.grammarly.com/blog/when-to-spell-out-numbers/

Reply:
The problem here is that it actually is  2 single quotes  ... So you suggest `Double single quotes` .. right?

Reply:
> The problem here is that it actually is 2 single quotes ... So you suggest `Double single quotes` .. right?

Yes, that's right.

Reply:
ok